### ISSUE TITLE
Keyboard Shortcut: "Ignore Check" does not work and Tooltip is wrong

### ISSUE THREAD

**Issue by anatolb** [WeblateOrg/weblate#2034]
### Steps to reproduce
1. open Page with String with failing check
2. Press CTRL+ I + Number of failing Check
3. Hover over number of failing check

### Actual behaviour
Firefox:  Bookmark sidebar opens, Tooltip says "Alt+I then <Number of check>"
Chrome and IE: Doesn't work either

### Expected behaviour
Check should be ignored
Shortcut stated in Tooltip should match with documentation

### Proposed solution
Use Alt+I consistently (worksOnMyMachine in Firefox, Chrome, IE)
```
--- a/weblate/static/loader-bootstrap.js
+++ b/weblate/static/loader-bootstrap.js
@@ -1062,7 +1062,7 @@ $(function () {
     Mousetrap.bindGlobal(['ctrl+i', 'command+i'], function(e) {});
     for (var i = 1; i < 10; i++) {
         Mousetrap.bindGlobal(
-            ['ctrl+i ' + i, 'command+i ' + i],
+            ['alt+i ' + i, 'command+i ' + i],
             function(e) {
                 return false;
             }
@@ -1083,7 +1083,7 @@ $(function () {
                 );

                 Mousetrap.bindGlobal(
-                    ['ctrl+i ' + key, 'command+i ' + key],
+                    ['alt+i ' + key, 'command+i ' + key],
                     function(e) {
                         $this.find('.close').click();
                         return false;
```

### Server configuration
 * Weblate weblate-2.19.1
 * Python 2.7.12
 * Django 1.11.2
 * six 1.10.0
 * social-auth-core 1.3.0
 * social-auth-app-django 1.2.0
 * django-appconf 1.0.2
 * Translate Toolkit 2.3.0
 * Whoosh 2.7.4
 * defusedxml 0.4.1
 * Git 2.7.4
 * Pillow (PIL) 1.1.7
 * dateutil 2.5.3
 * lxml 3.6.1
 * django-crispy-forms 1.6.1
 * compressor 2.1.1
 * djangorestframework 3.8.2
 * user-agents 1.1.0
 * pytz 2016.6.1
 * pyuca N/A
 * python-bidi 0.4.0
 * pyLibravatar N/A
 * PyYAML 3.12
 * Mercurial 3.8.4
 * Database backends: django.db.backends.mysql

**Comment by nijel** [comment]
It was changed to this to avoid some collisions with browser shortcuts, see #1291. The docs is wrong here.

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).

**Comment by anatolb** [comment]
Hi. 

maybe I am missing something but in my oppinion the tooltip was correct!

<span style="color:red">**CTRL+I**</span> doesnt work in Firefox, because it collides with the "Open Bookmarks Sidebar" shortcut (https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_bookmarks)!

<span style="color:green">**ALT+I**</span> instead works well in Firefox, Chrome, Internet Explorer, Edge on my Windows client.

Cheers,
Anatol

**Comment by reloxx13** [comment]
Since 1 or 2 versions the shortcut tooltip is broken :(

![grafik](https://user-images.githubusercontent.com/14855001/71068320-c39e2400-2176-11ea-9300-c4d9cb89f1e6.png)

Currently Weblate 3.9.1, tested in firefox and chrome.

**Comment by vanous** [comment]
Can we perhaps please choose a different shortcut for the dismiss action? `ctrl-o`, `ctrl-enter`, `mouse click` :cry:, `ctrl-enter`. the mouseclick breaks the flow significantly. As a repo owner, i can help to lessen the burned on translators by (translating to my language first and while doing it) clicking "Dismiss for all languages", but unfortunately, with the same ergonomics issue. I have disabled `weblate.checks.same.SameCheck` as people wanted (to my surprise) to re-start using desktop poeditor... Just my small feedback. Thank you for all your work.

**Comment by nijel** [comment]
Any proposals for that? We might get inspired by what GitHub uses: https://help.github.com/en/github/getting-started-with-github/keyboard-shortcuts

**Comment by vanous** [comment]
Sorry for later reply. Interestingly enough, when typing into very this issue comment text box, Ctrl+i is not taken by the default browser action, so perhaps we do not need to get inspired by what they use but how they do it :)

**Comment by nijel** [comment]
It could be a bug in Mousetrap library we use. It doesn't seem to be much maintained these days. Maybe it's time to look for alternative.

**Comment by nijel** [comment]
For example, https://wangchujiang.com/hotkeys/ seems to catch Ctrl+I just fine.

**Comment by julen** [comment]
As I've been able to confirm, Mousetrap catches <kbd>Ctrl/Cmd+i</kbd> just fine. The problem in Weblate is that both <kbd>Ctrl/Cmd+i</kbd> **and** <kbd>Ctrl/Cmd+i \<n\></kbd> shortcuts are assigned, and the latter, when being more specific, takes over.

In other words, when pressing <kbd>Ctrl/Cmd+i</kbd> Mousetrap doesn't have a way to know (without waiting) if this is pressed on its own, or as a first step of <kbd>Ctrl/Cmd+i \<n\></kbd>.

I can see two solutions here:
* Cancel browser's <kbd>Ctrl/Cmd+i</kbd> in vanilla JS without Mousetrap.
* Choose a shortcut for which the initial combination before the number doesn't clash with existing browser functionality across platforms.

By the way, I notice the event handler references `.dismiss-single`, which is not present since 4d963bc so that's broken as well.

**Comment by nijel** [comment]
I've fixed the broken selector in 9bfb91bc0f.

I think it makes sense to choose different shortcut. It used to be Alt+i, but that had different problem #1291

**Comment by julen** [comment]
Which one do you have in mind? Note how <kbd>Alt+i \<n\></kbd> needs exactly the same fix as <kbd>Ctrl/Cmd+i \<n\></kbd>, and if you say <kbd>Alt+i</kbd> combinations have some additional issues, I don't see what the improvement is of choosing that to be frank.

I'd suggest to keep the existing shortcut and cancel out <kbd>Ctrl/Cmd+i</kbd>. I was able to confirm it works well in the tests I've done.

**Comment by nijel** [comment]
I do not have suggestion right now and certainly it was not meant to go back to Alt+i. If canceling out addresses the issue, then I think it's good way to go. Thanks for taking care of this!

**Comment by julen** [comment]
I fixed the problem in #3891 based on the discussion above. I also notice the Machine Translation shortcut probably suffers from the same issue because both <kbd>mod+m</kbd> and <kbd>mod+m \<N\></kbd> are assigned via Mousetrap.
